kogito-codegen-modules/kogito-codegen-processes-integration-tests/src/test/java/org/kie/kogito/codegen/tests/PublishEventIT.java

```diff
@@ -207,11 +225,14 @@ public void testBasicUserTaskProcess() throws Exception {
         uow.end();
         assertThat(processInstance.status()).isEqualTo(ProcessInstance.STATE_COMPLETED);
         events = publisher.extract();
-        assertThat(events).hasSize(2);
-        body = assertProcessInstanceEvent(events.get(0), "UserTasksProcess", "UserTasksProcess", 2);
-        assertThat(body.getNodeInstances()).hasSize(2).extractingResultOf("getNodeType").contains("HumanTaskNode", "EndNode");
-        assertThat(body.getNodeInstances()).extractingResultOf("getTriggerTime").allMatch(v -> v != null);
-        assertThat(body.getNodeInstances()).extractingResultOf("getLeaveTime").allMatch(v -> v != null);
+
+        assertProcessInstanceEvent(events.get(3), "UserTasksProcess", "UserTasksProcess", 2);
```

Comment:
@elguardian this test is failing here... I can fix it with this:
```suggestion
        assertNodeInstanceEvent(events.get(3), processInstance.id(), "End Event 1", 1);
```